app/styles/app.scss

```diff
@@ -201,3 +201,8 @@ button .material-icons {
     padding: 15px;
     font-size: 16px;
 }
+
+.file-browser-list>div>div>div>div{
```

Comment:
There must be a better way to override this, if its using treebeard cells have their own classes. Can you check this?